public/index.html

```diff
@@ -5,6 +5,8 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
 
+    <meta name="Description" content="You want to learn a new skill? Steer away from doubts? Looking for someone to discuss? Coding Coach is here to connect you with mentors worldwide!">
```

Comment:
"Coding Coach is here to connect mentors and mentees around the world, for free. We believe mentorship should be accessible to all, regardless of circumstance." Thanks!

Reply:
One of the greatest Github's features 😄

```suggestion
    <meta name="Description" content="Coding Coach is here to connect mentors and mentees around the world, for free. We believe mentorship should be accessible to all, regardless of circumstance.">
```